src/constants/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const TODOS_URL = 'https://jsonplaceholder.typicode.com/todos';
+export const USERS_URL = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
давай через одну ссылку, в нужном месте дописывай `todos / users`
```suggestion
export const API_URL = 'https://jsonplaceholder.typicode.com/';
```